prov/efa/src/rdm/rxr_env.c

```diff
@@ -281,39 +247,55 @@ void rxr_env_initialize()
 /**
  * @brief Fetch the value of the environment variable FI_EFA_USE_DEVICE_RDMA
  *
- * This function fetches the change in the value of FI_EFA_USE_DEVICE_RDMA
- * if any, after it was defined and set in rxr_env_define_use_device_rdma() 
- * 
+ * This function fetches the value of FI_EFA_USE_DEVICE_RDMA from the
+ * user's environment.
+ *
+ * Environments with non-boolean (1, true, yes, on) values for the variable
+ * will print to stderr and abort the program.
+ *
+ * If environment variable is not set, return RDMA device support as
+ * is the default setting for the variable.
+ *
  * @return		0 - If FI_EFA_USE_DEVICE_RDMA is set to 0/false/no/off
- * 			1 - If FI_EFA_USE_DEVICE_RDMA is set to 1/true/yes/on
- * 			If FI_EFA_USE_DEVICE_RDMA is not set, then the value
- * 			returned from rxr_env_get_default_use_device_rdma()
- * 			will be the return value of this function.
+ *			1 - If FI_EFA_USE_DEVICE_RDMA is set to 1/true/yes/on
  */
-int rxr_env_get_use_device_rdma()
+bool rxr_env_get_use_device_rdma()
 {
-	int ret, use_device_rdma;
+	int ret;
+	int param_val;
```

Comment:
Don't we need a default value for `param_val`? which is used when FI_EFA_USE_DEVICE_RDMA is not set.